content/2019-02-20-newsletter-15.md

```diff
@@ -0,0 +1,191 @@
++++
+title = "The Embedded Working Group Newsletter - 15"
+date = 2019-02-20
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the fifteenth newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [users.rust-lang.org], [on twitter], or [on reddit]!
+
+[users.rust-lang.org]: https://example.org/#TODO
```

Comment:
Maybe comment these out? Otherwise we will get feedback that the links are broken.